### ISSUE TITLE
Date Picker accessibility: LocalizedControlType is 'edit' instead of 'calendar'

### ISSUE THREAD

**Issue by ronak-92** [OfficeDev/office-ui-fabric-react#1688]
### Bug Report

- __Package version(s)__: latest

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:
When inspecting the properties of date picker control following is observed. Same is observed on fabric date picker sample in official site.

![image](https://cloud.githubusercontent.com/assets/9554299/25662564/98870300-3032-11e7-9de5-d7cdd3a73d52.png)

#### Actual behavior:

Localized control type is 'edit'

#### Expected behavior:

Localized control type should be 'control'
